locales/en/moves.json

```diff
@@ -109,16 +109,11 @@
     "single_moves_no_results_pending": "No single requests pending review",
     "single_moves_no_results_approved": "No single requests have been approved",
     "single_moves_no_results_rejected": "No single requests have been rejected",
+    "move_from": "Move from",
     "move_to": "Move to",
     "earliest_move_date": "Earliest date of travel",
     "created_at": "Sent on",
-    "move_type": "Move type",
-    "filter": {
-      "proposed": "pending review",
-      "approved": "approved",
-      "rejected": "rejected",
-      "total": "sent for review"
-    }
```

Comment:
Are we sure those are now unused? Just checkin'.

Reply:
Nope, the unified dashboard at the root of the application uses the existing translations for these.